src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheckTest.java

```diff
@@ -54,7 +56,11 @@ public void testBadOverrideFromObject() throws Exception {
             "53:5: " + getCheckMessage(MSG_KEY_ANNOTATION_MISSING_OVERRIDE),
         };
 
-        verify(checkConfig, getPath("InputMissingOverrideBadOverrideFromObject.java"), expected);
```

Comment:
can we just update one line 
from
`verify(checkConfig, getPath("InputMissingOverrideBadOverrideFromObject.java"), expected);`
to
`verifyWithBddParser(checkConfig, getPath("InputMissingOverrideBadOverrideFromObject.java"), expected);`

so that all logic will be hidden inside `verifyWithBddParser`

it would much easier for you and other contributors to update only one line in test cases

Reply:
sure